addons/mail/static/src/components/notification_alert/notification_alert.xml

```diff
@@ -4,9 +4,18 @@
     <t t-name="mail.NotificationAlert" owl="1">
         <div class="o_NotificationAlert">
             <t t-if="env.isMessagingInitialized()">
-                <center t-if="isNotificationBlocked" class="o_notification_alert alert alert-primary">
-                    Odoo Push notifications have been blocked. Go to your browser settings to allow them.
-                </center>
+                <div t-if="isNotificationBlocked" class="o_notification_alert alert alert-primary text-center">
+                    Odoo Push notifications have been blocked. Go to your browser settings.
```

Comment:
The spec says the message should be different for safari, and also contain link for some browser? Wasn't it not possible to make a link?